scripts/run-shellcheck.sh

```diff
@@ -28,6 +28,13 @@ is_bash() {
 
 while IFS= read -r -d $'' file; do
     if is_bash "$file"; then
-        shellcheck -x -W0 -s bash "$file" || continue
+	shellcheck -x -W0 -s bash "$file"
+	rc=$?
+	if [ "${rc}" -eq 0 ]
+	then
+            continue
+        else
```

Comment:
The indentation looks a little wonky here

Reply:
Silly tabs! Will fix.